<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="hasilLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="grayButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Graysclase</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperasi_Titik">
    <property name="title">
     <string>Operasi Titik</string>
    </property>
    <addaction name="actionBrightness"/>
    <addaction name="actionSimple_Contrast"/>
    <addaction name="actionContrast_Stretching"/>
    <addaction name="actionNegative_Image"/>
    <addaction name="actionBiner_Image"/>
    <addaction name="actionHistogram_Grayscale"/>
    <addaction name="actionRGB_Histogram"/>
    <addaction name="actionHistogram_Equalization"/>
   </widget>
   <widget class="QMenu" name="menuOperasi_Geometri">
    <property name="title">
     <string>Operasi Geometri</string>
    </property>
    <widget class="QMenu" name="menuRotasi">
     <property name="title">
      <string>Rotasi</string>
     </property>
     <addaction name="action_45_Derajat"/>
     <addaction name="action45_Derajat"/>
     <addaction name="action_90_Derajat"/>
     <addaction name="action90_Derajat"/>
     <addaction name="action180_Derajat"/>
    </widget>
    <widget class="QMenu" name="menuMenu_Resize">
     <property name="title">
      <string>Menu Resize</string>
     </property>
     <addaction name="actionZoom_in"/>
     <addaction name="actionZoom_out"/>
     <addaction name="actionSkewed_Image"/>
    </widget>
    <addaction name="actionTranslasi"/>
    <addaction name="menuRotasi"/>
    <addaction name="menuMenu_Resize"/>
    <addaction name="actionCrop"/>
    <addaction name="actionAdd"/>
    <addaction name="actionSub"/>
   </widget>
   <widget class="QMenu" name="menuD">
    <property name="title">
     <string>D</string>
    </property>
    <addaction name="actionKonvolusi"/>
    <addaction name="actionMeanFilter"/>
    <addaction name="actionGaussian_Filter"/>
    <addaction name="actionSharping"/>
    <addaction name="actionMedian_Filtering"/>
    <addaction name="actionMax_Filtering"/>
   </widget>
   <addaction name="menuOperasi_Titik"/>
   <addaction name="menuOperasi_Geometri"/>
   <addaction name="menuD"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="actionSimple_Contrast">
   <property name="text">
    <string>Simple Contrast</string>
   </property>
  </action>
  <action name="actionContrast_Stretching">
   <property name="text">
    <string>Contrast Stretching</string>
   </property>
  </action>
  <action name="actionNegative_Image">
   <property name="text">
    <string>Negative Image</string>
   </property>
  </action>
  <action name="actionBiner_Image">
   <property name="text">
    <string>Biner Image</string>
   </property>
  </action>
  <action name="actionHistogram_Grayscale">
   <property name="text">
    <string>Histogram Grayscale</string>
   </property>
  </action>
  <action name="actionRGB_Histogram">
   <property name="text">
    <string>Histogram RGB</string>
   </property>
  </action>
  <action name="actionHistogram_Equalization">
   <property name="text">
    <string>Histogram Equalization</string>
   </property>
  </action>
  <action name="actionTranslasi">
   <property name="text">
    <string>Translasi</string>
   </property>
  </action>
  <action name="action_45_Derajat">
   <property name="text">
    <string>-45 Derajat</string>
   </property>
  </action>
  <action name="action45_Derajat">
   <property name="text">
    <string>45 Derajat</string>
   </property>
  </action>
  <action name="action_90_Derajat">
   <property name="text">
    <string>-90 Derajat</string>
   </property>
  </action>
  <action name="action90_Derajat">
   <property name="text">
    <string>90 Derajat</string>
   </property>
  </action>
  <action name="action180_Derajat">
   <property name="text">
    <string>180 Derajat</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionSkewed_Image">
   <property name="text">
    <string>Skewed Image</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionadd">
   <property name="text">
    <string>add</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionSub">
   <property name="text">
    <string>Sub</string>
   </property>
  </action>
  <action name="actionKonvolusi">
   <property name="text">
    <string>Konvolusi</string>
   </property>
  </action>
  <action name="actionMeanFilter">
   <property name="text">
    <string>Mean Filter</string>
   </property>
  </action>
  <action name="actionGaussian_Filter">
   <property name="text">
    <string>Gaussian Filter</string>
   </property>
  </action>
  <action name="actionSharping">
   <property name="text">
    <string>Sharping</string>
   </property>
  </action>
  <action name="actionMedian_Filtering">
   <property name="text">
    <string>Median Filtering</string>
   </property>
  </action>
  <action name="actionMax_Filtering">
   <property name="text">
    <string>Max Filtering</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
